shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/main/java/org/apache/shardingsphere/data/pipeline/api/metadata/model/PipelineTableMetaData.java

```diff
@@ -89,14 +89,14 @@ public PipelineColumnMetaData getColumnMetaData(final String columnName) {
     /**
      * Judge whether column is unique key or not.
      *
-     * @param columnIndex column index
+     * @param columnIndex the first column is 1, the second is 2, ...
      * @return true if the column is unique key, otherwise false
      */
     public boolean isUniqueKey(final int columnIndex) {
-        if (columnIndex >= columnNames.size()) {
+        if (columnIndex > columnNames.size()) {
             return false;
         }
```

Comment:
1, Could we add `uniqueKey` in `PipelineColumnMetaData`? It might could replace `primaryKey`.

2, Could we remove `isUniqueKey` method in `PipelineTableMetaData`?